README.md

```diff
@@ -1,8 +1,42 @@
+# Nkiruka Odu . Jarvis Consulting
 
+Interfacing between business and technology interests me a lot. I obtained a master's degree in computer science from the African University of Science and Technology. I have about 3 years of working experience as an assistant program coordinator at Nile University; where I worked with other team members to prepare proposals, projects, and organize other academic activities. This experience got me exposed to great working culture and best industrial practices which improved my creative thinking, communication,  and documentation skills. Working as a junior developer at Jarvis exposed me to the entire spectrum of software development processes, having the great opportunity to work in an Agile/Scrum environment where I collaborated with other team members to develop business solutions using technologies like Python, Pandas, Numpy, SQL, Bash, and Docker. Currently, I have a passion to work in an environment where I can actively collaborate with other team members to develop efficient business information technology systems by fostering the active use of emerging technologies and techniques to help them achieve strategic business goals. Therefore, I am currently seeking a full-time role as a Business System Analyst (BSA).  I am proficient in Python, SQL, data modeling, data analytics, and statistics. I am an excellent communicator, together with good documentation and presentation skills, which is required to liaise well with the stakeholders, especially during requirement elicitation and analysis. Learning new skills and how to apply them to solve real-world problems motivates me a lot!.
+
+## Skills
+
+**Business Skills:** Interpersonal, Communication, Documentaion/Presentation, Requirement Elicitation, Agile/Scrum
+
+**Technical Skills:** RDBMS/SQL, Data Modeling/Mapping, Data Analytics, Python, Excel, Google Cloud Platform, Bash, Git, Docker
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NkirukaOdu](https://github.com/jarviscanada/jarvis_data_eng_NkirukaOdu)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NkirukaOdu/tree/master/linux_sql)]: Linux Cluster Resource Monitoring App is used by the linux cluster agents to collect hardware specifications and resource usage information.  I prepared the business requirement documents (BRD) to show the basic requirements, the project objectives, the need statement, project scope, the programming language, identified the  stakeholders and performed elicitation through open interview, drew the project timeline/milestone/deadline and the cost benefit analysis. I also prepared the System requirement document (SRD) to show the  behaviour of the system, the functional and non-functional requirement, created data models and ER-diagram to show how different entities are related and the use case diagram to show the functional behaviours .
```

Comment:
**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NkirukaOdu/tree/master/linux_sql)]: Linux Cluster Resource Monitoring App is used by the Linux cluster agents to collect hardware specifications and resource usage information. I prepared the business requirement documents (BRD) to show the basic requirements, the project objectives, the need statement, project scope, the programming language, identified the stakeholders and performed elicitation through open interviews, drew the project timeline/milestone/deadline and the cost-benefit analysis. I also prepared the System requirement document (SRD) to show the system's behaviour and the functional and non-functional requirements. I created data models and ER diagrams to show how different entities are related and the use case diagram to show the functional behaviours.